presto-product-tests/conf/docker/common/jdbc_db.yml

```diff
@@ -24,3 +24,12 @@ services:
       MYSQL_PASSWORD: swarm
       MYSQL_ROOT_PASSWORD: swarm
       MYSQL_DATABASE: test
+
+  sqlserver:
+    hostname: sqlserver
+    image: 'microsoft/mssql-server-linux'
+    ports:
+      - '1433:1433'
+    environment:
+      ACCEPT_EULA: Y
```

Comment:
How does it work with our apache-licensed software and our automation? Can we legally use it on Travis?

This should be AT LEAST mentioned in README.